aunt-leahs-app/src/components/textField.jsx

```diff
@@ -29,10 +29,14 @@ const TextInput = ({ title, size, onChange }) => {
                     placeholder={' ' + title}
                     margin='none'
                     fullWidth='True'
-                    onChange={onChange} />
+                    onChange={onChange}
+                    type={type}
+                    value={value} />
```

Comment:
onChange should probably be a call to a function that updates the state. The value should probably reflect the state. And the type needs to be a valid HTML5 input type. 

Please see here: https://material-ui.com/api/text-field/
and here: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#Form_%3Cinput%3E_types

Let me know if you have any questions!

Reply:
I was thinking the page/component(i.e. adminLoginPage or form) that has textField as a child component should be the one holding the state instead of the textField component itself. This way the 'parent' component/page passes in handleState function as props.onChange to handle the change and likewise for props.value. The handleState function will then set value of the textField(via props.value) to the state if need be. To access the value from the textField, the handleState function in the parent component would have an 'events' parameter(example: handleState(events){do whatever}; ) and 'events.target.value' would give the user input value. 

As for the type, its default value is 'text'. So if the parent component does not specify a type(which I think will/should be for most cases) it will just be your regular textField. I added this parameter for special cases where the parent component might want to specify the type (I.e. password or date).  

Let me know if I need to change it.

Reply:
That should all be fine, but you should add a comment for each of the props, so that anyone who wants to use this component will quickly and easily know what it expects.

Reply:
noted